src/app/body/task-details/edit-page/edit-page.component.html

```diff
@@ -33,7 +33,7 @@ <h4 class="modal-title pl-3" id="exampleModalCenterLabel">Edit Task</h4>
                                 *ngIf="editTask.SprintNumber!=currentSprintNumber">Active Sprint</span>
                             <span id="Backlog" class="p-2 md-2" (click)="showBacklog()"
                                 *ngIf="editTask.SprintNumber!=backlogSprintNumber">
-                                Backlog </span>
+                                Backlog</span>
```

Comment:
made a small change here to draw your attention to the earlier change made in "Active Sprint" and "Backlog" button which got masked due to earlier changes done by "auto format in VS Code"